sites/processingmagazine.com/server/templates/index.marko

```diff
@@ -75,29 +75,6 @@ $ const adSlots = ({ aliases }) => ({
       </div>
     </div>
 
-    <div class="row">
-      <div class="col-lg-4 mb-block">
-        <shared-website-section-list-block alias=alias>
-          <@query-params limit=4 skip=5 />
```

Comment:
Since this was removed, the load more skip value needs to be reduced by 4 (from 9 to 5)

Reply:
pushed